### DISCUSSION TITLE
Bezeichnungen vertauscht?

### DISCUSSION THREAD

**Discussion by Kenaschon** [dewenni/ESP_Buderus_KM271#76] [General]
Hallo dewenni,

sind hier die Bezeichnungen vertauscht?

![temperatur](https://github.com/dewenni/ESP_Buderus_KM271/assets/47242804/a98de546-11e7-4983-92bf-6474646c7494)

Muß das nicht so

9:Winter| 10..30:Schwelle (°C) | 31:Sommer

Grüße
Manfred

**Comment by dewenni** [comment]
Hi,

nein das passt schon so.
Der Parameter bedeutet "Sommer ab". Man stellt also ein, ab welcher Außentemperatur die Steuerung auf Sommer umschaltet.
Je geringer diese Temperatur ist, umso früher wird auf Sommer umgeschaltet. Je höher diese Temperatur ist, desto später wird von Winter auf Sommer umgeschaltet.
Der reguläre Wertebereich ist von 10°C bis 30°C.
Die obere und untere Grenze haben hier eine besondere Bedeutung.
Bei 9°C wird fix Sommer eingestellt und bei 31° wird fix Winter eingestellt.

Sinngemäß:
wenn man bei 9° schon auf Sommer schalten will, dann kann es immer Sommer sein
Wenn man erst bei 31° auf Winter schalten will, dann soll es auch immer Winter sein.

**Comment by Kenaschon** [comment]
Ok,

das ist aber schon ein wenig um die Ecke denken.

Danke für die Info.

Grüße
